test/e2e/features/cat-c/98-welshfulljourney.feature

```diff
@@ -0,0 +1,30 @@
+@catC
+Feature: Full Welsh end to end journey
+
+Scenario: Examiner completes a passed welsh test with no faults
+Given I am logged in as "desexaminerw" and I have a test for "Miss Luella Lowery"
```

Comment:
Can you indent this file correctly please :)

Reply:
fixed
'@full_smoke' tag added